CHANGELOG.md

```diff
@@ -1,20 +1,21 @@
 # Aries Cloud Agent Python Changelog
 
-## 0.12.3rc0
+## 0.12.3
 
-### December 16, 2024
+### December 17, 2024
 
-A patch release to add address a bug found in the Linked Data Verifiable Credential handling for multi-tenant holders. The bug was fixed in the main branch, [PR 3391 - BREAKING: VCHolder multitenant binding](https://github.com/openwallet-foundation/acapy/pull/3391), and with this release is backported to 0.12 Long Term Support branch. Prior to this release, holders credentials received into a tenant wallet were actually received into the multitenant admin wallet.
+A patch release to add address a bug found in the Linked Data Verifiable Credential handling for multi-tenant holders. The bug was fixed in the main branch, [PR 3391 - BREAKING: VCHolder multitenant binding](https://github.com/openwallet-foundation/acapy/pull/3391), and with this release is backported to 0.12 Long Term Support branch. Prior to this release, holder credentials received into a tenant wallet were actually received into the multi=tenant admin wallet.
```

Comment:
Not sure what happened with multi=tenant :/

Reply:
My terrible keyboard that I can’t get used to...